From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Miklos Szeredi <mszeredi@suse.cz>
Date: Tue, 26 Oct 2010 10:50:32 +0000
Subject: [PATCH] vfs: export do_splice_direct() to modules

Export do_splice_direct() to modules.  Needed by overlay filesystem.

Signed-off-by: Miklos Szeredi <mszeredi@suse.cz>

[OpenIRD: refreshed for linux-2.6.31.6-rt19; no functional change.
 Source: linux-fsdevel "[PATCH 4/8 v4] vfs: export do_splice_direct() to modules",
 https://marc.info/?l=linux-fsdevel&m=128809035423071 ]
---
 fs/splice.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fs/splice.c b/fs/splice.c
index 73766d2..42e8058 100644
--- a/fs/splice.c
+++ b/fs/splice.c
@@ -1269,6 +1269,7 @@ long do_splice_direct(struct file *in, loff_t *ppos, struct file *out,
 
 	return ret;
 }
+EXPORT_SYMBOL(do_splice_direct);
 
 static int splice_pipe_to_pipe(struct pipe_inode_info *ipipe,
 			       struct pipe_inode_info *opipe,
